packages/edit-site/src/components/header-edit-mode/page-content-breadcrumbs/index.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect, useDispatch } from '@wordpress/data';
+import {
+	__experimentalHStack as HStack,
+	Icon,
+	Flex,
+	Button,
+	FlexItem,
+	__experimentalSpacer as Spacer,
+} from '@wordpress/components';
+import {
+	page as pageIcon,
+	chevronRight as chevronRightIcon,
+	sidebar as sidebarIcon,
+} from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+import { useEntityRecord } from '@wordpress/core-data';
+
+/**
+ * Internal dependencies
+ */
+import { store as editSiteStore } from '../../../store';
+
+export default function PageContentBreadcrumbs() {
+	const { hasPageContentLock, context } = useSelect(
+		( select ) => ( {
+			hasPageContentLock: select( editSiteStore ).hasPageContentLock(),
+			context: select( editSiteStore ).getEditedPostContext(),
+		} ),
+		[]
+	);
+
+	const { hasResolved: hasPageResolved, editedRecord: page } =
+		useEntityRecord( 'postType', context.postType, context.postId );
+
+	const { togglePageContentLock } = useDispatch( editSiteStore );
+
+	if ( ! hasPageResolved ) {
+		return null;
+	}
+
+	return hasPageContentLock ? (
+		<HStack
+			className="edit-site-page-content-breadcrumbs"
+			expanded={ false }
+			spacing={ 2 }
+		>
+			<Icon icon={ pageIcon } />
+			<div>{ page.title }</div>
+		</HStack>
+	) : (
+		<Flex
+			className="edit-site-page-content-breadcrumbs"
+			expanded={ false }
+			gap={ 1 }
+		>
+			<Button
+				className="edit-site-page-content-breadcrumbs__page-crumb"
+				icon={ pageIcon }
+				onClick={ () => togglePageContentLock() }
+			>
+				{ page.title }
+			</Button>
+			<Icon
+				className="edit-site-page-content-breadcrumbs__chevron"
+				icon={ chevronRightIcon }
+				size={ 16 }
+			/>
+			<FlexItem className="edit-site-page-content-breadcrumbs__template-crumb">
+				<Spacer padding={ 2 } marginBottom={ 0 }>
+					<HStack expanded={ false } spacing={ 2 }>
+						<Icon icon={ sidebarIcon } />
```

Comment:
We use the `layout` icon for templates in the view mode and lots of other places
![image](https://github.com/WordPress/gutenberg/assets/93301/47078b83-5e16-48fa-81fc-2341aaa47456)

It's the icon for the template entity type, shouldn't this use the `layout` icon rather than `sidebar`?